src/test/java/unit/Group.java

```diff
@@ -315,6 +294,32 @@ private static void expectListedMemberWithPolicies(
         "test user has the right policies");
   }
 
+  private void expectListedMemberWithPoliciesTableFormat(
+      String name, String email, GroupPolicy... policies) throws JsonProcessingException {
+    // call `terra group list-users` in table format
+    TestCommand.Result cmd = TestCommand.runCommand("group", "list-users", "--name=" + name);
+
+    // assert header is correct
+    assertTrue(cmd.stdErr == null || cmd.stdErr.isEmpty());
+    assertEquals(0, cmd.exitCode, "group list-user returned successfully");
+    String[] rows = cmd.stdOut.split("\\n");
+    String[] rowHead = rows[0].split("\\s+");
+    assertEquals("EMAIL", rowHead[0].trim().replace("\r", ""));
+    assertEquals("POLICIES", rowHead[1].trim().replace("\r", ""));
+
+    // assert email and policies are correct
+    boolean emailExists = false;
+    for (int i = 1; i < rows.length; i++) {
+      if (rows[i].contains(email)) {
+        emailExists = true;
+        for (var policy : policies) {
```

Comment:
Looks like you are asserting that every row must contain every policy. I don't think it works that way in the general case.

Reply:
I think the policy there is a parameter pass from the function. in this test, we test that one has 2 policies, so we pass 2 polices here and use for loop to iterate it.